Cargo.toml

```diff
@@ -8,3 +8,6 @@ edition = "2021"
 
 [dependencies]
 libc = "0.2"
+
+[build-dependencies]
+bindgen = "0.59.2"
```

Comment:
We want to have the generated Rust interface included in the project, so consumers of the library don't have to depend on bindgen + its dependencies.